packages/postgres-database/src/simcore_postgres_database/migration/versions/b60363fe438f_adds_column_to_use_scicrunch_alternative.py

```diff
@@ -19,16 +19,17 @@ def upgrade():
     # ### commands auto generated by Alembic - please adjust! ###
     op.add_column(
         "group_classifiers",
-        sa.Column("uses_scicrunch", sa.Boolean(), nullable=False, default=False),
+        sa.Column("uses_scicrunch", sa.Boolean(), nullable=True, server_default="0"),
```

Comment:
hmm why a 0? I see postgres can take it all. but from python it is a bit nicer to work with True/False no?

Reply:
with the server_default you need text